### ISSUE TITLE
Forcieren, dass Opfor zu Fuss geht

### ISSUE THREAD

**Issue by McDiod** [gruppe-adler/TvT_WITA2016.Altis#22]
@AdlerXiviD sagte:

* Grundsätzlich öfter markieren wenn Opfor in Fahrzeug ist? Forcieren, dass Opfor zu Fuss geht

**Comment by Salbei2500** [comment]
Die Frage ist doch, ist dies notwendig? 
Blufor hat mittlerweile 5 Helis zur Verfügung(1 davon schwer Bewaffnet). Muss man da noch stärker zu Fuß Action des Agenten forcieren?

**Comment by AdlerXiviD** [comment]
Ja, weil das ewige rumfahren das Spiel langweilig macht. Unabhängig davon wurde der Vorschlag gemacht, bevor ein bewaffneter Heli eingeführt wurde. Dieser wird btw das Spiel ziemlich schnell beenden wenn ein guter Pilot fliegt :D

**Comment by Salbei2500** [comment]
Wäre nicht die Möglichkeit für Blufor, Nagelbänder auszulegen besser?
Es ist nicht so OP wie ein Bewaffneter Heli, stopt zuverlässig den Agenten und gibt Blufor ne Super chance für einen Hinterhalt!

**Comment by McDiod** [comment]
Der bewaffnete Heli hat mMn nichts mit dieser Idee zu tun, sondern ist nur dafür da, dass das Spiel nicht zu lang werden kann (z.B. weil Blufor nicht aus ihrem Startheli auf Boote schießen kann).

Nagelbänder find ich ne witzige Idee. Sie müssten aber entweder relativ gut sichtbar oder nur in geringer Menge verfügbar sein, damit Blufor sie nicht einfach auf jede Straße spammen kann.

**Comment by Salbei2500** [comment]
[Spike Strips](https://i.ytimg.com/vi/iwQA_GAD0qk/hqdefault.jpg?sqp=-oaymwEWCKgBEF5IWvKriqkDCQgBFQAAiEIYAQ==&rs=AOn4CLB1pic3BPKQJO07TZYccGHS1a5Kqg) Wäre sowas ne Option? Sehr beliebt bei Altis Live :trollface: 

Zum Heli, sind die zwei Racketen notwendig?

**Comment by nomisum** [comment]
Statt Nagelband könnte man auch direkt Barrikaden nehmen oder sone Rot-Weiß angemalten Holzgestelle. Letztere würden vllt. nem Quad den Reifen anditschen. Erstere hätten die selbe Wirkung wie n Nagelband: Agent fährt drumrum.

**Comment by Salbei2500** [comment]
Du musst das ja nur gut Positionieren, hinter einer Scharfen kurve in einem Ort oder so 🗡 
Es soll ja auch nicht OP sein, der Agent soll auch ne chance haben dem Auszuweichen und so ein nagelband rendert erst 2-3m vor dir bei 150+ kmh
